stats/api/views.py

```diff
@@ -0,0 +1,77 @@
+from django.http import JsonResponse
+from django.shortcuts import render
+from django.views import View
+from rest_framework import generics, permissions, filters
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from silver.api.filters import TransactionFilter, SubscriptionFilter
+from silver.models import Transaction, Subscription
+from stats.api.serializers import TransactionStatsSerializer
+from stats.stats import Stats
+from datetime import datetime, timedelta
+
+
+class HomeView(View):
+    def get(self, request, *args, **argv):
+        return render(request, 'charts.html', {})
+
+
+class SubscriptionStatsChart(APIView):
+    authentication_classes = []
+    permission_classes = []
+
+    def get(self, request, format=None):
+        labels = ["plan1", "plan2", "plan3", "plan4", "plan5"]
+        default_items = [1, 6, 9, 3, 1]
+        default_items2 = [2, 2, 3, 9]
+        data = {
```

Comment:
This looks like it's been written for a specific kind of client consumer, which is not a good idea. But first I'll wait for you to reveal the purpose of this class.